src/Stripe.net/Services/Customers/CustomerUpdateOptions.cs

```diff
@@ -24,16 +24,16 @@ public class CustomerUpdateOptions : BaseOptions, IHasMetadata
         public long? Balance { get; set; }
 
         [JsonProperty("coupon")]
-        public string Coupon { get; set; }
+        public AnyOf<string, EmptyParam> Coupon { get; set; }
 
         [JsonProperty("default_source")]
-        public string DefaultSource { get; set; }
+        public AnyOf<string, EmptyParam> DefaultSource { get; set; }
 
         [JsonProperty("description")]
-        public string Description { get; set; }
+        public AnyOf<string, EmptyParam> Description { get; set; }
 
         [JsonProperty("email")]
-        public string Email { get; set; }
+        public AnyOf<string, EmptyParam> Email { get; set; }
```

Comment:
Does it make sense to support this on string since you can simply pass an empty string today and it just works?

Reply:
That's certainly debatable. Is this:

```c#
var options = new CustomerUpdateOptions { Email = EmptyParam.Empty };
```

better/clearer than this:

```c#
var options = new CustomerUpdateOptions { Email = "" };
// or even
var options = new CustomerUpdateOptions { Email = string.Empty };
```
?

In stripe-java we do define an overload with `EmptyParam` even for strings, but we disallow empty strings, which we don't do in stripe-dotnet.

Let's discuss this with @cjavilla-stripe and @richardm-stripe next week.